KomaMRICore/src/simulation/Bloch/BlochSimulationMethod.jl

```diff
@@ -76,25 +82,28 @@ It gives rise to a rotation of `M0` with an angle given by the efective magnetic
     a part of the complete Mag vector and it's a part of the initial state for the next
     precession simulation step)
 """
-function run_spin_excitation!(p::Phantom{T}, seq::DiscreteSequence{T}, sig::AbstractArray{Complex{T}},
-    M::Mag{T}, sim_method::Bloch) where {T<:Real}
+function run_spin_excitation!(
+    p::Phantom{T},
+    seq::DiscreteSequence{T},
+    sig::AbstractArray{Complex{T}},
+    M::Mag{T},
+    sim_method::Bloch,
+) where {T<:Real}
     #Simulation
-    for s ∈ seq #This iterates over seq, "s = seq[i,:]"
```

Comment:
Revert removal of comment.

Reply:
Resolved in [665f6f6](https://github.com/JuliaHealth/KomaMRI.jl/pull/184/commits/665f6f6188ad0c2fc9b9511de3e70f91337b78a2)